### PR TITLE
W-10875483: A.2.2 Implementation of the minimum set of attributes

### PR DESCRIPTION
**Pull request by fsgonz**

### CODE REVIEW

File: core/src/main/java/org/mule/runtime/core/internal/profiling/tracing/event/span/CoreEventSpanCustomizer.java [hunk 1]
```diff
@@ -22,8 +28,21 @@ public interface CoreEventSpanCustomizer {
    *
    * @param coreEvent the {@link CoreEvent} to resolve the span name from.
    * @param component the {@link Component} to resolve the span name from.
-   * @return
+   * @return the name
```
Comment (IvanAndresFritzler) [inline core/src/main/java/org/mule/runtime/core/internal/profiling/tracing/event/span/CoreEventSpanCustomizer.java:31]: ... of the span?

File: core/src/main/java/org/mule/runtime/core/internal/profiling/tracing/export/OpentelemetrySpanExporter.java [hunk 2]
```diff
@@ -59,6 +59,10 @@ public OpentelemetrySpanExporter(Tracer tracer, EventContext eventContext,
 
   @Override
   public void export(InternalSpan internalSpan) {
+    for (Map.Entry<String, String> entry : internalSpan.attributesAsMap().entrySet()) {
```
Comment (IvanAndresFritzler) [inline core/src/main/java/org/mule/runtime/core/internal/profiling/tracing/export/OpentelemetrySpanExporter.java:62]: Is this ok to be here or should we add the attributes to the OTEL span as soon as they are added to our internal span?

**Review by IvanAndresFritzler — CHANGES_REQUESTED** [review]

File: core/src/main/java/org/mule/runtime/core/internal/profiling/tracing/export/OpentelemetrySpanExporter.java (hunk 2 above)
Comment (fsgonz) [reply core/src/main/java/org/mule/runtime/core/internal/profiling/tracing/export/OpentelemetrySpanExporter.java]: In practical terms, it is the same as the export in opentelemetry happens when you set the end, but I've changed it for future developments.

**Review by IvanAndresFritzler — DISMISSED** [review]

**Comment by fsgonz** [comment]
--validate

**Comment by fsgonz** [comment]
--validate

**Comment by fsgonz** [comment]
--validate

**Review by IvanAndresFritzler — DISMISSED** [review]

**Review by IvanAndresFritzler — APPROVED** [review]

**Pull request merged**